src/OmniSharp.LanguageServerProtocol/LanguageServerHost.cs

```diff
@@ -62,7 +62,7 @@ public class LanguageServerHost : IDisposable
                 .WithInput(input)
                 .WithOutput(output)
                 // initializeParams from the client won't be arriving yet, configure with app loglevel
-                .ConfigureLogging(GetLogBuilderAction(configureLogging, application.LogLevel))
+                .ConfigureLogging(AddLanguageProtocolLogging(application.LogLevel))
```

Comment:
previously there were two calls to `GetLogBuilderAction`, one here and one when building the composition (after initialize).  Both calls registered the lsp logger and so we got double logs.

I modified this to register the LSP logger initially (with default level) and then configure it on composition building (after initialize).